framework/Models/ROM.py

```diff
@@ -1171,9 +1171,9 @@ def __init__(self,runInfoDict):
       @ Out, None
     """
     Dummy.__init__(self,runInfoDict)
-    self.initializationOptionDict = {}          # ROM initialization options
-    self.amITrained                = False      # boolean flag, is the ROM trained?
-    self.supervisedEngine          = None       # dict of ROM instances (== number of targets => keys are the targets)
+    self.initializationOptionDict = {'NumThreads': runInfoDict['NumThreads']}         # ROM initialization options
```

Comment:
why num threads and not batchsize? NumThreads is most of the time 1 in RAVEN...shouldn't be batchsize?